dotnet-471-dev-pack/test/lib/habitat.ps1

```diff
@@ -0,0 +1,53 @@
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Private Functions START
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+function habitat::_install_pester {

```

Comment:
This would be a common function and should live in the `[ROOT]/bin/` directory instead of being inside a specific plan.
We should maybe check on Slack whether it would be better to ensure Pester is installed by:
 A. having a core plan for it
 B. having a common `habitat::install_pester` function, or
 C. Since it's just a single PS module, put "You need Pester" in the README, and do not take responsibility for its installation (though at some point we will need to make sure it's available on the CI agents).

Reply:
Resolved; habitat.ps1 has been removed altogether.  After refactoring the test.ps1 so that it can run in different contexts like ci and builder, the test.ps1 has been slimmed down and habitat.ps1 removed.